src/components/KeywordInput/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { CloseIcon } from "@assets/svgs";
+
+interface KeywordInputProps {
+  label: string;
+  placeholder: string;
+  keywords: string[];
+  onAddKeyword: (keyword: string) => void;
+  onDeleteKeyword: (keyword: string) => void;
+  validateKeyword?: (keyword: string) => boolean;
+  errorMessage?: string;
+  isRequired?: boolean;
+}
+
+const KeywordInput: React.FC<KeywordInputProps> = ({
+  label,
+  placeholder,
+  keywords,
+  onAddKeyword,
+  onDeleteKeyword,
+  validateKeyword,
+  errorMessage,
+  isRequired = false,
+}) => {
+  const [inputValue, setInputValue] = useState<string>("");
+  const [isValid, setIsValid] = useState<boolean>(true);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setInputValue(value);
+
+    if (value.trim() === "") {
+      setIsValid(true);
+    }
+  };
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === "Enter" && inputValue.trim()) {
+      e.preventDefault();
+      const keyword = inputValue.trim();
+
+      if (validateKeyword && !validateKeyword(keyword)) {
+        setIsValid(false);
+        return;
+      }
+
+      if (keywords.includes(keyword)) {
+        setIsValid(false);
+        return;
+      }
+
+      onAddKeyword(keyword);
+      setInputValue("");
+      setIsValid(true);
+    }
+  };
+
+  return (
+    <div className="flex flex-col">
+      <label className="flex items-center">
+        <p className="mr-[6px] text-md font-semibold text-title">{label}</p>
+        <p className="text-sm font-regular text-body3">
+          [{isRequired ? "필수" : "선택"}]
+        </p>
+      </label>
+      <input
+        placeholder={placeholder}
+        value={inputValue}
+        onChange={handleInputChange}
+        onKeyPress={handleKeyPress}
```

Comment:
[변경 요청]
```suggestion
        onKeyDown={handleKeyDown}
```
onKeyPress 없어지고 onKeyDown으로 바꼈어요. 이렇게 바꿔주세요